### DISCUSSION TITLE
How to make the site accessible public?

### DISCUSSION THREAD

**Discussion by knchris** [globaleaks/globaleaks-whistleblowing-software#4560] [Q&A] [answered]
I need assistance the site can be accessed on the LAN but I cannot access it publicly i have a public IP addresses pointing to a domain name but i cannot access it i need assistance

**Comment by gianlucagilardi** [comment, marked as answer]
1) have you configured port forwarding correctly to publish ports 80 and 443  on the server on the public IP address?
2) have you configured the domain name (hostname) in the GL network settings to match the domain name you want to use to access the platform from the WAN?

**Comment by knchris** [comment]
> 1. have correctly configured port forwarding correctly to publish ports 20 and 443  on the server on the public IP address?
> 2. have you configured the domain name (hostname) in the GL network settings to march the domain name you want to use to access the platform from the WAN?

let me try that and i revert

**Comment by knchris** [comment]
> > 1. have correctly configured port forwarding correctly to publish ports 20 and 443  on the server on the public IP address?
> > 2. have you configured the domain name (hostname) in the GL network settings to march the domain name you want to use to access the platform from the WAN?
> 
> let me try that and i revert

this has worked thanks so much

**Comment by evilaliv3** [comment]
Thank you for confirming this! I'm going to close the ticket then.
